lib/catalog/order_item_sanitized_parameters.rb

```diff
@@ -1,22 +1,16 @@
-require 'mustache'
-
```

Comment:
This was removed here but not replaced in the `OrderItemRuntimeParameters` file, is it just being included somehow implicitly?

Reply:
In my test I found out no need to explicitly require this gem. TBH I don't know where it is implicitly included.